packages/store-ui/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "@vtex/tsconfig",
   "include": ["src"],
   "compilerOptions": {
+    "baseUrl": "src",
```

Comment:
Can we add `forceConsistentCasingInFileNames:true` ? Since you're touchin on it?